operator/src/main/resources/Operator.properties

```diff
@@ -170,6 +170,9 @@ WLSDO-0020=Online WebLogic configuration updates complete \
 WLSDO-0021=DomainUID ''{0}'' server ''{1}'' does not have a port available for the operator to send REST calls. \
   The default listen port and SSL port are disabled, the admin port is not configured and there is no channel with admin privileges.\
 
+WLSDO-0022=The common mount configured by 'spec.serverPod.commonMount' in the domain resource does not define any containers. \
+  The common mount must specify at least one container.
```

Comment:
serverPod can also scope to the cluster level or even the pod level, so 'spec.serverPod.commonMount', so:
- Change 'The common mount' to 'A common mount'
- Change 'spec.serverPod.commonMount' to 'serverPod.commonMount'.